torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -444,6 +446,7 @@ def add_choices(
         trans_w=False,
         input_indices=None,
         epilogue_creator: Optional[Callable[[ir.Buffer], ir.Pointwise]] = None,
+        is_woq_gemm=False,
```

Comment:
scale epilogue is applied, and this case doesn't have a bias. I may end up removing it, though, especially since int4 WoQ GEMM requires more special-casing, and I'd rather add a new argument for it than int8 WoQ GEMM.

Thanks!